component/src/main/wso2/sap/Module.md

```diff
@@ -36,14 +36,11 @@ It has full IDoc and experimental BAPI support. It uses the SAP JCO library as t
 Follow the steps below to build the Ballerina SAP endpoint from the source code:
 
 1. Get a clone or download [the source](https://github.com/wso2-ballerina/module-sap).
-2. Create a lib folder in the module-sap directory.
-3. Download the sapidoc3.jar and sapjco3.jar middleware libraries from the SAP support portal and copy those 
-   libraries to the module-sap/lib directory.
-4. Navigate to the folder `module-sap` directory and execute the following Maven command:
+2. Navigate to the folder `module-sap` directory and execute the following Maven command:
     
         mvn clean install
-5. Extract the `/component/target/wso2-sap-<version>.zip` distribution. 
-6. Execute either of the install.{sh/bat} scripts to install the module.
+3. Extract the `/component/target/wso2-sap-<version>.zip` distribution. 
+4. Execute either of the install.{sh/bat} scripts to install the module.
```

Comment:
At which point the SAP libs are copied? Are we asking the user to provide the location of the libs when the the installation script runs?

Reply:
I have added  in the https://github.com/wso2-ballerina/module-sap/blame/master/README.md#L11.

No need to provide the location for lib when the installation script runs.

Reply:
We shouldn't copy a set of artifacts thorough the script and request user to copy another set manually. The purpose of having the script is to hide the details from the user. So, it would be better if we modify the script to request the downloaded SAP lib location as a parameter, and copy the jars through the script.

Reply:
As per the Rajkumar, customer didn't agree to provide the location of the libs when the the installation script runs.